### ISSUE TITLE
Archlinux AUR kamaki error: BAD REQUEST badRequest (API endpoint not found)

### ISSUE THREAD

**Issue by epidrome** [grnet/kamaki#119]
I am getting the following errors

`
➜  ~ kamaki image list                     
(400) BAD REQUEST badRequest (API endpoint not found)
➜  ~ kamaki file list /images                                          
  1.61GiB arch.diskdump
➜  ~ kamaki image register --name=arch --location=/images/arch.diskdump
Nonexistent image file location pithos://user-id/images/arch.diskdump
|  BAD REQUEST badRequest (API endpoint not found)\n
|  Does the image file arch.diskdump exist at container images ?
|  To get current user id
|    kamaki user info
➜  ~ kamaki user info
id: user-id
`

In short, kamaki reports that the image file is there but next it is reported as missing during registration

I am not sure if this is this a bug or if something is missing in my configuration? i have installed kamaki through arch linux AUR and using a kamakirc file

**Comment by saxtouri** [comment]
IMO, it looks like a bug in the way kamaki reports errors. Part of the error is missleading.

The "API endpoint not found" error shows that kamaki cannot execute the "kamaki image list" command at all. Seems like you are using a kamaki version incobatible to the cloud API version. 

Do you use kamaki with ~okeanos? If yes, you probably need to update kamaki to the latest version (1.16).  To check the kamaki version:
```
$ kamaki --version
```

**Comment by epidrome** [comment]
you are right! on my archlinux AUR installation the reported kamaki version is 0.13 and it produces the above error, so i have to use a python virtualenv with the latest version 0.16, which works fine with okeanos
in the meantime i had the chance to run the okeanos suite (snf+kamaki) on a debian machine and it worked without error
thanks for the prompt reply!

**Comment by axilleas** [comment]
I updated the Arch Linux package to 0.16 https://aur.archlinux.org/packages/kamaki/.
